SourceryTests/Parsing/FileParser + MethodsSpec.swift

```diff
@@ -56,7 +56,7 @@ class FileParserMethodsSpec: QuickSpec {
                 }
 
                 it("extracts class method properly") {
-                    expect(parse("class Foo { class func foo() }")).to(equal([
+                    expect(parse("class Foo { class func foo() {} }")).to(equal([
```

Comment:
what if its a protocol definition and it doesn't have the `{}` ?

Reply:
it is covered with other tests